packages/api/src/platforms/vtex/clients/index.ts

```diff
@@ -1,15 +1,18 @@
+import type { Context, Options } from '..'
 import { VtexCommerce } from './commerce'
+import { Profile } from './profile'
 import { IntelligentSearch } from './search'
-import type { Context, Options } from '..'
 
 export type Clients = ReturnType<typeof getClients>
 
 export const getClients = (options: Options, ctx: Context) => {
   const search = IntelligentSearch(options, ctx)
   const commerce = VtexCommerce(options, ctx)
+  const profile = Profile(options, ctx)
 
   return {
     search,
     commerce,
+    profile,
```

Comment:
Now we've been discussing whether to keep everything inside commerce as it is or refactor it into separate folders for each service. In this specific scenario, I thought it would be a good idea to introduce a new one since we will also be using it for My Account.

Reply:
We decided to not do. Keep everything in `commerce` 🙇